### ISSUE TITLE
Error "Could not load DLL file linohone.dll"

### ISSUE THREAD

**Issue by viniciusoliveira12** [bedefaced/sipdotnet#14]
Hi, I have a problem with the published version of the application, it is a windows forms application C #, when I try to make a call it gives me the error that "DLL file 'linphone.dll can not be loaded. please your support one more time, thank you very much for the help.

**Comment by bedefaced** [comment]
Are 'linphone.dll' and other required dlls placed in the directory of your application EXE?
Doesn't you run your program on Windows XP (see #13)?

**Comment by viniciusoliveira12** [comment]
Good morning, yes the dlls are already in the same folder of the executable, my project does not run on XP. I create a project published, and that publish users install on the machines. Any other idea of what it could be? Thank you.

**Comment by bedefaced** [comment]
Could you please provide full error message?

**Comment by viniciusoliveira12** [comment]
Hello, I managed to achieve the error, you were correct friend, I had to have the linphone dlls, in the folder inside User \ AppData ... where the installation is when done by publish. The only problem I have now is when I make a call, and the person does not answer, and rejects it immediately, gives me a mistake when this situation happens.

**Comment by viniciusoliveira12** [comment]
Of course, Operating System is Windows 7, 64-bit architecture, but my project I have in "Any CPU".

**Comment by vijayparmar1992** [comment]
I'm getting same error as @voliveiradev .
Property Node (Arg :1) Error accessing property Linphone.Factory.TopResourcesDir, (System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
	Inner Exception: System.DllNotFoundException: Unable to load DLL 'liblinphone': The specified module could not be found. (Exception from HRESULT: 0x8007007E)) <append><b>System.DllNotFoundException</b> in Account Config and Creation.vi

**Comment by qq5050595531** [comment]
anyone fix it ?

**Comment by farazp** [comment]
I have this error to 
Unable to load DLL 'linphone' or one of its dependencies: The specified module could not be found. (0x8007007E)
